templates/_rows_and_columns.html

```diff
@@ -1,16 +1,40 @@
-<table>
+<table id="dbresults">
     <thead>
         <tr>
-            {% for column in display_columns %}<th scope="col">{{ column }}</th>{% endfor %}
+            <th scope="col">Link</th>
+            <th scope="col">First name</th>
+            <th scope="col">Middle initial</th>
+            <th scope="col">Last name</th>
+            <th scope="col">Organization</th>
+            <th scope="col">YTD gross earnings</th>
         </tr>
     </thead>
     <tbody>
     {% for row in display_rows %}
-        <tr>
+        <tr class="{{ loop.cycle('odd', 'even') }} fancyfloat">
             {% for cell in row %}
-                <td>{{ cell.value }}</td>
+                {% if cell.column == 'Link' %}
```

Comment:
I haven't tested this in Jinja, but I believe you can write `if cell.column in {'Link', 'first_name', ...}` instead of doing multiple if statements.

Reply:
Or you could only test for ytd_gross_earnings, format that one, but give each one a class like col-Link, col-first_name, … and hide the classes you don't want with CSS.

Reply:
yup, fixed with the second solution